test.js

```diff
@@ -120,6 +120,66 @@ test('skip throwing when using reject option in sync mode', t => {
 	t.is(exitCode, 2);
 });
 
+test('execa() with .kill after it with SIGKILL should kill cleanly', async t => {
+	const subprocess = execa('node', ['fixtures/no-killable'], {
+		stdio: ['ipc']
+	});
+
+	await pEvent(subprocess, 'message');
+
+	subprocess.kill('SIGKILL');
+
+	const {signal} = await t.throwsAsync(subprocess);
+	t.is(signal, 'SIGKILL');
+});
+
+// Windows doesn't support sending signals. No re-emulates them down to SIGKILL
```

Comment:
> No re-emulates them down to SIGKILL

I don't understand what this is trying to say. It should also end in a `.`

Reply:
```suggestion
// `SIGTERM` cannot be caught on Windows, and it always also abort the process (like `SIGKILL` on Unix).
// Therefore, this feature and those tests do not make sense on Windows. 
```

Reply:
Thanks, fixed: bd37113e03cddea242b2cb084bc56ef0eca88221